### ISSUE TITLE
dataloader 

### ISSUE THREAD

**Issue by FrankWuuu** [yassouali/CCT#65]
in dataloader` drop_last=True `?
Otherwise, 
`dataloader = iter(zip(cycle(self.supervised_loader), self.unsupervised_loader)) `
might get different size

**Comment by yassouali** [comment]
hi @FrankWuuu 

drop_last=True just removes the last batch if the number of its example < batch size, so at most you might lose 9 examples (for a batch of 10) at a given iteration, but then in the next you will see these examples since we train with a shuffle, i don't think it'll have any significant impact; tho it shouldn't be used for validation
